src/components/Header/AboutList.tsx

```diff
@@ -0,0 +1,13 @@
+import { AboutListData } from '../../constants/header';
```

Comment:
AboutList and StoreList are the same. Add props to one components and remove other.